internal/pkg/deploy/cloudformation/stack/static_site_integration_test.go

```diff
@@ -64,6 +66,15 @@ func TestStaticSiteService_TemplateAndParamsGeneration(t *testing.T) {
 					Name: &envName,
 				},
 			}
+
+			// Create in-memory mock file system.
```

Comment:
dumb q! where is the mocked `fs` used in this test case?

Reply:
resolved offline